client/cli/build.rs

```diff
@@ -0,0 +1,21 @@
+use std::{env, fs::File, process::Command};
+fn main() {
+    let prev_hash_var = env::var("GIT_HASH");
+    if let Ok(prev_hash) = prev_hash_var {
+        if prev_hash.len() < 4 || &prev_hash[0..4] != "fedi" {
+            //A user wants to override the hash
+            println!("cargo:rustc-env=GIT_HASH={}", prev_hash);
+            return;
+        }
+    }
+    let git_hash = if File::open("./../../.git/HEAD").is_ok() {
+        let output = Command::new("git")
+            .args(&["rev-parse", "HEAD"])
+            .output()
+            .unwrap();
+        String::from_utf8(output.stdout).unwrap()
+    } else {
+        String::from("fedif7069afbe457afa1d2683a099c7af48a523c")
```

Comment:
Some comment about why is this hardcoded for the confused passer-by would be good to have.

Also we only want something catchy in here (visually), it doesn't have to be `fedi` related.